themes/blist/layouts/partials/header.html

```diff
@@ -33,6 +35,19 @@
             </svg>
           </span>
         </a>
+        {{ else }}
+        <p class="dropdown-switcher flex items-center">
```

Comment:
I think <div> will be better than <p> in this case because there is no text paragraph here.

Reply:
Sorry, i don't see with what i should replaced paragraphs in your message, so i replaced it with divs:)) if thats not what you mean, tell me)

Reply:
Hah, Yeah I meat div))